edpm_ansible/roles/edpm_iscsid/meta/main.yml

```diff
@@ -29,10 +29,10 @@ galaxy_info:
   # https://galaxy.ansible.com/api/v1/platforms/
   #
   platforms:
-    - name: 'EL'
+    - name: CentOS
```

Comment:
According to [Galaxy APIs](https://galaxy.ansible.com/api/v1/platforms), there's no object with `name="CentOS"`. We should start using `'EL'`.